### PR TITLE
[FOR REVIEW] Add ZeroEx settler trades on Polygon

### PR DESCRIPTION
**Pull request by RantumBits**
### Contribution type
Please check the type of contribution this pull request is for:

- [x] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name* zeroex_polygon.settler_trades
- **Description:** [Detailed description of the new spell(s) and their purpose] decode 0x API trades on new settler contracts on polygon
- **Who are the new spell(s) for?** [Internal team or general community] both
- **How will the new spell(s) be used downstream?** [Description of downstream usage] added to dex_agg.trades
- **Implementation details:** [Information on how the spell(s) are implemented]  gets taker and maker amounts only, no fills level data for intermediary swaps
- **Test instructions:** [How to test the new spell(s)] - https://dune.com/queries/3921632
- **Related issue(s):** [Link to related issues, if any]

### CODE REVIEW

**Comment by RantumBits** [comment]
updated to match requested changes on #6425

File: dex/seeds/_project/zeroex/polygon/_schema.yml [hunk 1]
```diff
@@ -66,3 +66,11 @@ seeds:
         tx_from: varbinary
         tx_to: varbinary
         fills_within: int
+  - name: zeroex_polygon_settler_trades_sample
```
Comment (Hosuke) [inline dex/seeds/_project/zeroex/polygon/_schema.yml:69]:
```suggestion

  - name: zeroex_polygon_settler_trades_sample
```

File: dex/models/_projects/zeroex/polygon/zeroex_polygon_schema.yml [hunk 2]
```diff
@@ -266,3 +266,53 @@ models:
       - &protocol_fee_paid_eth
         name: protocol_fee_paid_eth
         description: "The protocol fee paid in ETH"
+  - name: zeroex_polygon_settler_trades
```
Comment (Hosuke) [inline dex/models/_projects/zeroex/polygon/zeroex_polygon_schema.yml:269]:
```suggestion

  - name: zeroex_polygon_settler_trades 
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits 

[Sample zeroex_polygon_settler_trades]()

**Comment by Hosuke** [comment]
Also here are some merge conflicts to resolve.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**